tests/end_to_end/doubles/irix-o2-mips1-out.c

```diff
@@ -2,7 +2,7 @@ f64 test(f64 arg0, s32 arg2, f64 arg4) {
     f64 temp_f0;
 
     temp_f0 = (((f64) arg2 * arg0) + (arg0 / arg4)) - 7.0;
-    if (!(temp_f0 < arg4)) {
+    if (temp_f0 >= arg4) {
```

Comment:
this looks wrong... float comparisons can't be negated like this, we're supposed to have logic that makes sure that doesn't happen